packages/trackerless-network/test/unit/StreamEntrypointDiscovery.test.ts

```diff
@@ -71,10 +74,21 @@ describe('StreamEntryPointDiscovery', () => {
 
     const fakeDeleteEntryPointData = async (_key: Uint8Array): Promise<void> => {}
 
+    const addNodesToStream = (layer1: MockLayer1, count: number) => {
+        range(count).forEach(() => {
+            layer1.addNewRandomPeerToKBucket()
```

Comment:
Would it make sense to rename `addNewRandomPeerToKBucket` e.g. to `addRandomNode()`?

Reply:
We might want to add tests where we add peers to the `neightborList` of `MockLayer1` nodes so it makes sense to differentiate between which struct is being used.